src/components/organisms/Results.vue

```diff
@@ -168,11 +208,12 @@ export default defineComponent({
     <FilterMenu />
     <v-data-table
       :headers="headers"
-      :items="searchResults"
+      :items="!props.regions ?searchResults :regionsList"
       :page.sync="page"
       :items-per-page.sync="itemsPerPage"
-      :server-items-length="searchResultsTotal"
+      :server-items-length="!props.regions ?searchResultsTotal :regionsTotal"
       :footer-props="{ itemsPerPageOptions }"
+      :class="props.regions && 'px-5'"
```

Comment:
I think this construct is not great for readability. Also, if `props.regions` is `false`, it will set `class=false`, and I'm not sure what the behavior of that is, or whether it's well-defined. I think it'd be better as

```suggestion
      :class="props.regions ? 'px-5' : ''"
```

It's possible this is defined behavior and an OK practice. If there are official docs showing it's the right way to do it, we should keep it.